nxdrive/gui/api.py

```diff
@@ -201,6 +202,34 @@ def get_transfers(self) -> List[Dict[str, Any]]:
 
         return result
 
+    def upload_is_direct_transfer(self, path, engine):
```

Comment:
Maybe a `upload.is_direct_transfer` is preferable to prefevent such code.

Reply:
the same than `is_direct_edit` ?

Reply:
Yes.